### PR TITLE
feature/style-adjustments

### PR DESCRIPTION
**Pull request by BrianWhitneyAI**
The purpose of this PR is to resolve UX improvements described in #94. 

# Requests
- [x] I think a 4px border radius on controls is subtle but add a nice refinement to the square corners. You can see this on the toggles at the top, etc.
- [x] I’m not fully committed to the “pill” shaped buttons yet, but you can see the adjustments I would suggest regardless of the border radius. They should have a max-width.
- [x] The size toggle meaning is not quite right. You are switching size, not lower/uppercase.
- [x] The toggles need to clearly show what is currently active. I also didn’t show this visually, but giving a 1px margin between the toggle buttons I think will help them stand out individually better but still feel like a grouping.
- [x] BE mindful of equal lef-right spacing. If they are equal, it will look much more refined.

## Changes 
- Increased view mode button curvature to 4px.
- Added margin to view mode buttons of 1px.
- Added rounded edge buttons for file detail buttons.
- Set max width of file detail buttons to 250px.
- Changed size toggle icon to both be capital "A" with font size difference.
- Set even spacing around the query sidebar to 1em.
- Changed colors of shadow for sidebar selection/deselection
- Changed color of sidebar selection/deselection icon. 
- Widened select/deselect region for query sidebar.
- Widened select/deselect region for file detail sidebar.

### CODE REVIEW

**Comment by BrianWhitneyAI** [comment]
fms_bug

Noticed this bug when messing with the width of the query sidebar collapse button. If the FIle detail window is closed and the window is minimized the bar is misaligned even though the button location and the query sidebar width use the same logic for their scaling.

**Review by SeanLeRoy — APPROVED** [review]

File: packages/core/components/FileDetails/FileDetails.module.css [hunk 1]
```diff
@@ -158,8 +159,9 @@
     border: none;
     color: var(--primary-text-color);
     height: 30px;
-    max-width: 50%;
+    max-width: var(--max-detail-button-width);
```
Comment (aswallace) [inline packages/core/components/FileDetails/FileDetails.module.css:162]: Could we also keep these aligned center? E.g., in `.file-actions {`, line 104 `justify-content: center;`

**Review by aswallace — APPROVED** [review]

**Comment by aswallace** [comment]
> Noticed this bug when messing with the width of the query sidebar collapse button. If the FIle detail window is closed and the window is minimized the bar is misaligned even though the button location and the query sidebar width use the same logic for their scaling

Suggested fix: 
In App.module.css, `.query-sidebar` (e.g., line 65): add `position: relative;`
In QuerySidebar.module.css, `.minimize-bar` (e.g., line 110): 
- remove the `left` attribute
- replace with `right: 0;`

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 1 above)
Comment (BrianWhitneyAI) [reply packages/core/components/FileDetails/FileDetails.module.css:162]: resolved in [43cb537](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/103/commits/43cb537ff521b7f5d90dbb268ae3525878022f9e)

**Comment by BrianWhitneyAI** [comment]
> > Noticed this bug when messing with the width of the query sidebar collapse button. If the FIle detail window is closed and the window is minimized the bar is misaligned even though the button location and the query sidebar width use the same logic for their scaling
> 
> Suggested fix: In App.module.css, `.query-sidebar` (e.g., line 65): add `position: relative;` In QuerySidebar.module.css, `.minimize-bar` (e.g., line 110):
> 
> * remove the `left` attribute
> * replace with `right: 0;`

resolved in [43cb537](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/103/commits/43cb537ff521b7f5d90dbb268ae3525878022f9e)

**Pull request merged**